chain/cosmos/upgrade/v1beta1/query.ts

```diff
@@ -661,7 +661,7 @@ export const QueryAuthorityResponse = {
 
   toJSON(message: QueryAuthorityResponse): unknown {
     const obj: any = {};
-    if (message.address !== "") {
```

Comment:
This changed the default value of fields so that we can generate proper json for EIP712, specially for array elements